packages/flutter/lib/src/material/button.dart

```diff
@@ -77,22 +111,33 @@ abstract class MaterialButton extends StatefulWidget {
   }
 }
 
+/// A state object for [MaterialButton].
+///
+/// Subclasses of [MaterialButton] should use a subclass of
+/// [MaterialButtonState] for their state objects.
 abstract class MaterialButtonState<T extends MaterialButton> extends State<T> {
+  /// Whether this button is in the process of potentially being pressed.
   bool highlight = false;
 
+  /// The height at which to place this button.
```

Comment:
Ian mentioned the problem with using "height" to describe material elevation elsewhere.  Perhaps what we need is a good description of the implications of elevation for elevationToShadow (or in the Material class) and links to that in places like this.